app/eth2wrap/eth2wrap.go

```diff
@@ -106,6 +107,27 @@ func (m multi) Address() string {
 	return m.clients[0].Address()
 }
 
+func NewWrapHTTP(eth2Svc eth2client.Service) (Client, error) {
+	httpCl, ok := eth2Svc.(*eth2http.Service)
+	if !ok {
+		return nil, errors.New("invalid eth2http client")
+	}
+
+	return httpWrap{httpCl}, nil
+}
+
+// httpWrap implements Client by wrapping eth2http.Service,
+// also implements experimental interfaces which are absent in eth2http.Service.
+type httpWrap struct {
+	*eth2http.Service
+}
+
+// SubmitBeaconCommitteeSubscriptions implements eth2exp.BeaconCommitteeSubscriptionsSubmitter.
+// TODO(dhruv): Implement this method when there's need to connect to external beacon node.
```

Comment:
you need to implement this now, otherwise beaconmock isn't going to be called. You need to add tests to ensure your code works, it currently doesn't.

Reply:
Done!